src/ucs/datastruct/usage_tracker.h

```diff
@@ -37,6 +37,8 @@ typedef struct {
     unsigned                           promote_thresh;
     /* Min score difference in order to remove an entry from promoted list [0-1] */
     double                             remove_thresh;
+    /* Min score required to promote an entry [0-1] */
+    double                             score_thresh;

```

Comment:
why needed?

Reply:
in order to prevent entries with low score to be promoted.